MAINTAINERS

```diff
@@ -591,7 +598,7 @@ made through a pull request.
 
 	[people.jfrazelle]
 	Name = "Jessie Frazelle"
-	Email = "jess@docker.com"
+	Email = "jess@linux.com"
```

Comment:
I almost put that, not going to lie, i debated over it ;)

On Thu, May 21, 2015 at 12:35 PM, Sebastiaan van Stijn <
notifications@github.com> wrote:

> In MAINTAINERS
> <https://github.com/docker/docker/pull/13391#discussion_r30838535>:
>
> > @@ -591,7 +598,7 @@ made through a pull request.
> >
> >  	[people.jfrazelle]
> >  	Name = "Jessie Frazelle"
> > -	Email = "jess@docker.com"
> > +	Email = "jess@linux.com"
>
> Be careful, they may take princess@docker from you!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/13391/files#r30838535>.
>